conn_go18.go

```diff
@@ -44,13 +45,30 @@ func (cn *conn) ExecContext(ctx context.Context, query string, args []driver.Nam
 
 // Implement the "ConnBeginTx" interface
 func (cn *conn) BeginTx(ctx context.Context, opts driver.TxOptions) (driver.Tx, error) {
-	if opts.Isolation != 0 {
-		return nil, errors.New("isolation levels not supported")
+	mode := ""
+
+	switch sql.IsolationLevel(opts.Isolation) {
+	case sql.LevelDefault:
+		// Don't touch mode: use the server's default
+	case sql.LevelReadUncommitted:
+		mode += " ISOLATION LEVEL READ UNCOMMITTED"
```

Comment:
Yeah, it's a bit cleaner without the concat.

As a note: go doesn't perform an alloc when concating with an empty string, see: https://play.golang.org/p/_Xp6RjpYZW